lib/suma/schema_document.rb

```diff
@@ -66,11 +66,13 @@ def output_extensions
       "xml"
     end
 
-    # #.gsub(/[\n\r]{2,}/, '')
+    # can't use 
+    # :lutaml-express-index: schemas; #{path_to_schema_yaml};
+    # because that kills any possibility of this file hyperlinking to other schemas
     def to_adoc(path_to_schema_yaml)
       <<~HEREDOC
         = #{@schema.id}
-        :lutaml-express-index: schemas; #{path_to_schema_yaml};
+        :lutaml-express-index: schemas; ../../schemas-srl.yml
```

Comment:
Yes we need to change this to the full set but the EXPRESS schema manifest file path is determined by the input to the `suma` command. We will need to replace the implementation of `path_to_schema_yaml` if we want hyperlinked schemas.